javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,12 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  let i = 0;
+  while (i < values.length) {
+    if (values[i] === null) {
+      values.splice(i, 1);
+    } else {
+      i++;
+    }
+  }
+  return values;
+}
```

Comment:
Again, we don't want to manipulate the `values` array directly, we just want to reference it. We should be utilizing a 2nd array to push values into. Check out my solution
```
function filterOutNulls(values) {
  const result = [];
  for (let i = 0; i < values.length; i++) {
    if (values[i] !== null) {
      result.push(values[i]);
    }
  }
  return result;
}
```

Reply:
Updated. Thank you for letting me know.